routes.js

```diff
@@ -0,0 +1,64 @@
+const Todo = require('./models/todo');
+
+module.exports = (server) => {
+  function todoNullCheck(todo, res, id) {
+    let a = true;
```

Comment:
Setting `a` here is pointless. Whenever you'd set `a` to false, just return false, and return true at the end